app/src/org/commcare/dalvik/application/CommCareApplication.java

```diff
@@ -1097,6 +1097,15 @@ public CommCareSessionService getSession() throws SessionUnavailableException {
         }
     }
 
+
+    public UserKeyRecord getRecordForCurrentUser() {
+        try {
+            return getSession().getUserKeyRecord();
+        } catch (SessionUnavailableException e) {
+            return null;

```

Comment:
Will this ever be ok? Not quite sure why this just wouldn't throw a SessionUnavailableException as part of the method signature